src/test/java/io/strimzi/kafka/bridge/http/AdminClientTest.java

```diff
@@ -136,4 +136,101 @@ void getTopicNotFoundTest(VertxTestContext context) throws Exception {
                 });
     }
 
+    @Test
+    void listPartitionsTest(VertxTestContext context) throws Exception {
+        String topic = "testListPartitions";
+        kafkaCluster.createTopic(topic, 2, 1);
+
+        JsonArray records = new JsonArray();
+        JsonObject json = new JsonObject();
+        json.put("value", "hello");
+        records.add(json);
+
+        JsonObject root = new JsonObject();
+        root.put("records", records);
+
+        // send a message and wait its delivery to make sure the topic has been made visible to the brokers
+        CompletableFuture<Boolean> producer = new CompletableFuture<>();
+        producerService()
+                .sendRecordsRequest(topic, root, BridgeContentType.KAFKA_JSON_JSON)
+                .sendJsonObject(root, ar1 -> {
+                    context.verify(() -> {
+                        assertThat(ar1.succeeded(), is(true));
+                        producer.complete(true);
+                    });
+                });
+        producer.get(TEST_TIMEOUT, TimeUnit.SECONDS);
```

Comment:
because you are using this way of making the topic visible to the brokers in different tests, it could be better to factor out a common method (to use in the other tests you already added as well). In this way, while reading test we are not distracted from lines of code that are not so much related to the test itself (even if they are useful for having the test working).

Reply:
ok